README.md

```diff
@@ -2,12 +2,12 @@
   <br>
   <img src="https://storage.googleapis.com/golden-wind/github/omni/omni.png" alt="Omni Logo" width="100">
   <br>
-  Omni for <a href="https://www.mozilla.org/firefox/">Firefox</a>
+  Omni for <a href="https://www.mozilla.org/firefox/">Firefox</a> and <a href="https://www.thunderbird.net/pt-BR/">Thunderbird</a>
```

Comment:
Shouldn't we have instructions on how to install it on Thunderbird? Are the same install steps from Firefox?

Reply:
The unique diferrence of install in Firefox and Thunderbird is: in Thunderbird we not have the address bar, then the user needs to access the Add-ons page from menu. I add in `INSTALL.md` a little section talking about this step on Thunderbird.